File: ConstDest.cpp
#include<iostream>
using namespace std;

inline void msg() { cout << "Welcome to C++" << endl ;}

class student{
    private:
        string fullname;
        int rollnum;
        double sempercentage;
        string collegename;
        int collegecode;
    public:
        student(){}
        student(string name,int roll,string college,int code,double percentage)
        {
            fullname = name;
            rollnum = roll;
            collegename = college;
            collegecode = code;
            sempercentage = percentage;

        }
        ~student(){}
        void printstudentdetails()
        {
            cout << "Name : " << fullname         << endl
            << "Rollno : " << rollnum << endl
            << "College : " << collegename << endl
            << "College code : " << collegecode << endl
            << "Sem percentage : " << sempercentage   << endl;
        }
       
        
};

int main()
{
    student s1("Naga",59,"Mvgr",33,99);

      s1.printstudentdetails();
   
}

File: week4/ConstDest.java
import java.util.Scanner;
class Student{
    String fullname;
    int rollnum;
    double sempercentage;
    String collegename;
    int collegecode;
    Student(){}
    Student(String name,int roll,String college,int code,double percentage)
    {
        fullname = name;
        rollnum = roll;
        collegename = college;
        collegecode = code;
        sempercentage = percentage;

    }
    void printstudentdetails()
    {
        System.out.println("The student details:");
        System.out.println("Name :" +fullname);
        System.out.println("Roll no :" +rollnum);
        System.out.println("College name :" +collegename);
        System.out.println("College code :" +collegecode);
        System.out.println("Sem percentage:" +sempercentage);
    }
}
public class ConstDest {
    public static void main(String[] args) {
        Scanner input  = new Scanner(System.in);

        System.out.print("Enter your name :");
        String name = input.nextLine();

        System.out.print("Enter your rollno :");
        int roll = input.nextInt();
        input.nextLine();
        System.out.print("Enter your college name :");
        String collname = input.nextLine();

        System.out.print("Enter your college code :");
        int code = input.nextInt();

        System.out.print("Enter your percentage :");
        double sem = input.nextDouble();

        Student s = new Student(name,roll,collname,code,sem);

        s.printstudentdetails();

        input.close();
    }
    
}
